YARP devel with YARP_USE_OPENCV set to OFF does not compile

With the following error: 
~~~
[ 91%] Built target YARP_OS
[ 91%] Built target YARP_init
[100%] Built target YARP_sig
Scanning dependencies of target yarpdatadumper
[100%] Building CXX object src/yarpdatadumper/CMakeFiles/yarpdatadumper.dir/main.cpp.o
/Users/traversaro/src/something-superbuild/isaac/YARP/src/yarpdatadumper/main.cpp:305:9: error: member initializer 't0' does not name a non-static data member or base class
        t0(0.0)
        ^~~~~~~
1 error generated.
make[3]: *** [src/yarpdatadumper/CMakeFiles/yarpdatadumper.dir/main.cpp.o] Error 1
make[2]: *** [src/yarpdatadumper/CMakeFiles/yarpdatadumper.dir/all] Error 2
make[1]: *** [src/yarpdatadumper/CMakeFiles/yarpdatadumper.dir/rule] Error 2
make: *** [yarpdatadumper] Error 2
~~~

--- a/src/yarpdatadumper/main.cpp
+++ b/src/yarpdatadumper/main.cpp
@@ -301,8 +301,7 @@ public:
         saveData(_saveData),
         videoOn(_videoOn),
         videoType(_videoType),
-        closing(false),
-        t0(0.0)
+        closing(false)
     {
         infoFile=dirName;
         infoFile+="/info.log";
@@ -311,6 +310,7 @@ public:
         dataFile+="/data.log";
 
     #ifdef ADD_VIDEO
+        t0 = 0.0;
         transform(videoType.begin(),videoType.end(),videoType.begin(),::tolower);
         if ((videoType!="mkv") && (videoType!="avi"))
         {
@@ -341,9 +341,6 @@ public:
     bool threadInit() override
     {
         oldTime=Time::now();
-        cumulSize=0;
-        counter=0;
-        closing=false;
 
         finfo.open(infoFile.c_str());
         if (!finfo.is_open())

